docs/terminal/amberleyoakey.md

```diff
@@ -11,92 +11,97 @@
 
 ## Airspace
 ### AMB TCU
-#### Restricted Areas
-By default, AMB APP owns all of the R625 Restricted Areas, as detailed below:
+**AMB APP** owns the airspace contained within:
 
 - R625A (`A015`-`A085`)  
 - R625B (`A025`-`A085`)  
 - R625C (`A045`-`A085`)  
-- R625D (`A085`-`F210`)  
-
-### Optional, as required
-AMB APP can negotiate further airspace releases from surrounding ENR sectors to facilitate planned military operations of the following Restricted Areas:
-
-- R650A (`A050`-`A100`)  
-- R650B (`A050`-`F600`)  
-- R662A (`A050`-`F600`)  
-- R662B (`A050`-`F600`)  
-- R671A (`F125`-`F245`)  
-- R671B (`F240`-`F600`)
-
-!!! note
-    See [VATPAC Notams](https://vatpac.org/publications/notam){target=new} for active NOTAMs which may affect military operations.
-
-#### Diagram
+- R625D (`A085`-`F210`)
 
 <figure markdown>
 ![AMB TCU Airspace Diagram](img/amtcu.png){ width="952" }
   <figcaption>AMB TCU Airspace Diagram</figcaption>
 </figure>
 
-### AMB ADC
-AMB ADC owns the Class C airspace in the AM CTR within 10nm of the YAMB ARP from `SFC` to `A015`. 
+**AMB ADC** owns the Class C airspace in the AM CTR from `SFC` to `A015`. 
 
 ### OK TCU
-#### Restricted Areas
-By Default OK APP owns all of the R625 Restricted Areas, detailed below:
+**OK APP** owns the airspace contained within:
 
 - R654A (`SFC`-`A065`)  
-- R6545B (`SFC`-`A065`)  
+- R654B (`SFC`-`A065`)  
 - R654C (`A065`-`F125`)  
 - R654D (`A065`-`F125`)  
 
-#### Diagram
-
 <figure markdown>
 ![OK TCU Airspace Diagram](img/oktcu.png){ width="1060" }
   <figcaption>OK TCU Airspace Diagram</figcaption>
 </figure>
 
-### OK ADC
-OK ADC owns the Class C airspace within 5 DME of the OK VOR from `SFC` to `A025`. This airspace is designed to facilitate circuit traffic.
+**OK ADC** owns the Class C airspace within 5 DME of the OK VOR from `SFC` to `A025`.
 
 
 ## YAMB Aerodrome
 ### Coded Departures (Fast Jets)
-Visual Departures are commonly requested by high performance military jets, and often processed in the form of a coded departure (e.g. BYRON7 departure), which can be found in the AD2 Sups Page for Amberley. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane avoiding civilian traffic inbound to Brisbane. 
-
+Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. BYRON1 departure), which can be found in the AD2 Sups Page for YAMB. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane, avoiding civilian traffic inbound to Brisbane. 
 
-Aircraft will make a visual departure in the circuit direction and fly overhead YAMB to begin tracking for their initial waypoint. 
+Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint. The following altitude constraints and tracking points are implied by the coded clearance. 
```

Comment:
Could it be possible to get whoever is running ACD to amend their flightplan with the relevant routing, so TCU can still have an idea of where fast departures are going?

Reply:
Yep, ADC page instructs them to